paasta_tools/setup_prometheus_adapter_config.py

```diff
@@ -58,7 +59,6 @@
 from paasta_tools.long_running_service_tools import METRICS_PROVIDER_PISCINA
 from paasta_tools.long_running_service_tools import METRICS_PROVIDER_PROMQL
 from paasta_tools.long_running_service_tools import METRICS_PROVIDER_UWSGI
-from paasta_tools.long_running_service_tools import MetricsProviderDict
```

Comment:
should the other autoscaling related imports from long_running_service_tools also be moved?